### PR TITLE
Replaced $ZSH to $ZSH_CUSTOM in Readme

### PR DESCRIPTION
**Pull request by idoo**

### CODE REVIEW

**Comment by MichaelAquilina** [comment]
Hi @idoo could you provide some context on what this change is meant to do?

What is `$ZSH_CUSTOM` and why is it better to use than `$ZSH`?

**Comment by idoo** [comment]
@MichaelAquilina because in the OMZSH docs there is an example on [customization](https://github.com/robbyrussell/oh-my-zsh/wiki/Customization) section contains $ZSH_CUSTOM instead of $ZSH + custom.

> Initially $ZSH_CUSTOM points to oh-my-zsh's custom directory.


I guess that path should use more constant as possible :)

Please feel free to close it if you prefer another way :)

File: README.rst [hunk 1]
```diff
@@ -69,12 +69,12 @@ Zgen_
...
 
-Copy this repository to ``$ZSH/custom/plugins``, where ``$ZSH`` is the
-root directory of oh-my-zsh:
+Copy this repository to ``$ZSH_CUSTOM/custom/plugins``, where ``$ZSH_CUSTOM`` 
+is the directory with custom plugins of oh-my-zsh:
 
 ::
 
-    git clone https://github.com/MichaelAquilina/zsh-you-should-use.git $ZSH/custom/plugins/you-should-use
+    git clone https://github.com/MichaelAquilina/zsh-you-should-use.git $ZSH_CUSTOM/plugins/you-should-use
```
Comment (MichaelAquilina) [inline README.rst]: not sure if maybe you forgot to push or if I was not clear, but could we include the link you provided in your previous comment here?

Maybe something like "See more details about custom plugins [here](link_url)"

Comment (idoo) [reply README.rst]: I got, what you mean. Thought that you about that I forgot to change that path near link after git clone :)

Comment (MichaelAquilina) [reply README.rst]: Thanks! The link seems broken though (Check https://github.com/MichaelAquilina/zsh-you-should-use/blob/86f40c68c41aec519f059d22316aa892bc8d6808/README.rst)

Comment (MichaelAquilina) [reply README.rst]: Just remembered why. The README uses restructured text. See this page for how to add links http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks

**Comment by MichaelAquilina** [comment]
Thanks for the PR @idoo !

**Pull request merged**